src/Api/User.hs

```diff
@@ -1,46 +1,41 @@
-{-# LANGUAGE DataKinds                  #-}
+{-# LANGUAGE DataKinds #-}
 {-# LANGUAGE GeneralizedNewtypeDeriving #-}
-{-# LANGUAGE TypeOperators              #-}
+{-# LANGUAGE TypeOperators #-}
 
 module Api.User where
 
-import           Control.Monad.Except
-import           Control.Monad.Reader        (ReaderT, runReaderT)
-import           Control.Monad.Reader.Class
-import           Data.Int                    (Int64)
-import           Database.Persist.Postgresql (Entity (..), fromSqlKey, insert,
-                                              selectFirst, selectList, (==.))
-import           Network.Wai                 (Application)
-import           Servant
-import           Servant.JS                  (vanillaJS, writeJSForAPI)
-
-import           Config                      (App (..), Config (..))
-import           Models
-import           Data.Text (pack)
-
-type UserAPI =
-         "users" :> Get '[JSON] [Entity User]
-    :<|> "users" :> Capture "name" String :> Get '[JSON] (Entity User)
-    :<|> "users" :> ReqBody '[JSON] User :> Post '[JSON] Int64
+import Control.Monad.Except
+import Control.Monad.Reader (ReaderT, runReaderT)
+import Control.Monad.Reader.Class
+import Data.Int (Int64)
+import Database.Persist.Postgresql
+       (Entity(..), (==.), fromSqlKey, insert, selectFirst, selectList)
+import Network.Wai (Application)
+import Servant
+import Servant.JS (vanillaJS, writeJSForAPI)
+
+import Config (App(..), Config(..))
+import Data.Text (Text)
```

Comment:
The convention the project follows is to have two blocks of imports:

```haskell
module ModuleName where

import ExternalLibrary
import Control.External.Bar

import InternalModule
import Models
```

External library modules are imported in the first block, and internal modules are imported in the second. In this case, the `Text` import should be in the top block.